www/js/app.js

```diff
@@ -2303,6 +2318,112 @@ function articleLoadedSW (iframeArticleContent) {
     params.isLandingPage = false;
 };
 
+/**
+ * Attaches popover trigger events to the given window
+ * @param {Window} win The window to which to attach popover trigger events
+ */
+function attachPopoverTriggerEvents (win) {
+    // The popover feature requires as a minimum that the browser supports the css matches function
+    // (having this condition prevents very erratic popover placement in IE11, for example, so the feature is disabled for such browsers)
+    if (appstate.wikimediaZimLoaded && params.showPopoverPreviews && 'matches' in Element.prototype) {
+        const iframeDoc = win.document;
+        if (iframeDoc) {
```

Comment:
nit: prefer `if (!iframedoc) return;`

In fact you have an if nested directly under another if. Consider:

```
const iframeDoc = win.document;
if (!iframeDoc || !appstate.wikimediaZimLoaded || !params.showPopoverPreviews  || !'matches' in Element.prototype) {
  return;
}
```

Reply:
Thanks, will check.